rocrate_validator/profiles/five-safes-crate/must/7_requested_workflow_run.ttl

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2025 eScience Lab, The University of Manchester
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+@prefix ro: <./> .
+@prefix ro-crate: <https://github.com/crs4/rocrate-validator/profiles/ro-crate/> .
+@prefix five-safes-crate: <https://github.com/eScienceLab/rocrate-validator/profiles/five-safes-crate/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix schema: <http://schema.org/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix validator: <https://github.com/crs4/rocrate-validator/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+five-safes-crate:RootDataEntityMentionsCreateAction
+    a sh:NodeShape ;
+    sh:name "RootDataEntity" ;
+    sh:targetClass ro-crate:RootDataEntity ;
+    sh:description "" ;
+
+    sh:sparql [
+        a sh:SPARQLConstraint ;
+        sh:name "RO-Crate must contain CreateAction" ;
+        sh:prefixes ro-crate:sparqlPrefixes ;
+        sh:select """
+        SELECT $this
+        WHERE {
+            $this a schema:Dataset .
+            FILTER NOT EXISTS { 
+                ?action a schema:CreateAction .
+            }
+        }
+        """ ;
+        sh:severity sh:Violation ;
+        sh:message "RO-Crate MUST contain at least one `CreateAction` entity." ;
+    ] ;
```

Comment:
Agreed on a call that this check "RO-Crate MUST contain at least one CreateAction entity." is redundant because of the other checks below (RDE MUST have `mentions` and `mentions` MUST reference a CreateAction). So this section can be removed